great-idea/index.html

```diff
@@ -16,7 +16,69 @@
 
   <body>
 
-    <!-- Copy and paste your HTML from the first UI project here -->
+    <div class="container">
+      <header>
+        <nav>
+          <a href="services.html">Services</a>
+          <a href="">Product</a>
+          <a href="">Vision</a>
+          <a href="">Features</a>
+          <a href="about.html">About</a>
+          <a href="">Contact</a> 
+        </nav>
+        <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+      </header>
+  
+  
+    <section>
+      <div class="mainheading-left">
+          <h1>Innovation<br>
+          On<br>
+          Demand</h1>
+          <a href="" alt="Get Started">Get Started</a>
+      </div>
+      <img class="logo2" src="img/header-img.png" alt="Image of a code snippet.">
+    </section>
+  
+        <div class="containerFirstSection">
+          <div class="FirstSectionItems"><h2>Features</h2>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+          </div>  
+          <div class="FirstSectionItems"><h2>About</h2>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+          </div>
+        </div>
+  
+        <div> 
+          <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+        </div> 
+  
+        <div class="containerThirdSection">
+          
+          <div class="ThirdSectionItems"><h2>Services</h2>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+          </div>
+          <div class="ThirdSectionItems"><h2>Product</h2>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+          </div>
+          <div class="ThirdSectionItems"><h2>Vision</h2>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+          </div>
+        </div>
+      
+        <footer>
+          <div class="footer-index-container">
+            <div class="contactInfo">
+              <h2>Contact</h2>
+              <p>123 Way 456 Street<br>
+              Somewhere, USA<br><br>
+              1 (888) 888-8888<br><br>
```

Comment:
```suggestion
              <a href="tel:18888888888">1 (888) 888-8888</a><br><br>
```
Putting this into an anchor tag and using 'tel:PHONE-NUMBER' will make this more semantic. Your user will know that this is a phone number, overall allowing for a better UX